examples/chrono_binding.cpp

```diff
@@ -6,6 +6,10 @@
 #include <format>
 
 #if __cpp_lib_chrono >= 201907L && __cpp_lib_format >= 201907L
+#define ENABLE_THIS_EXAMPLE
```

Comment:
what is the main different of this file's modification? it still doesn't work if `!(__cpp_lib_chrono >= 201907L && __cpp_lib_format >= 201907L)` and reverse

Reply:
We have to provide `int main()`, and `ENABLE_THIS_EXAMPLE` conditionally compiles main()'s body, see code further down.